fh-messaging/lib/fh_metrics_man.js

```diff
@@ -202,19 +202,65 @@ MetricsManager.prototype.deleteMetricsData = function(topic, date, callback) {
   messageMan = new fhmsg.Messaging(msgConfig, self.mLogger);
```

Comment:
@davidkirwan Do you know where this entire function is called from? "deleteMetricsData"

the original code was just doing a simple `var collectionName = topic + "_" + helpers.toYYYYMMDD(date);` and then dropping that collection. I wonder how often this function gets called. Does it get called for every topic?

Reply:
https://github.com/feedhenry/fh-messaging/blob/28e142848ba873c7861041d57ce33df06c658a82/fh-messaging/lib/shell/processone.sh#L54-L58

Yup it will get called for each topic.